### PR TITLE
Add iCubErzelli02

### PR DESCRIPTION
**Pull request by Nicogene**
This PR adds the robot `iCubErzelli02`

Please review code.

### CODE REVIEW

**Comment by traversaro** [comment]
According to https://icub-tech-iit.github.io/documentation/icub_versions/, iCub 2.5 with backpack is 2.5.5, but in https://robot-bazaar.iit.it/'s robot manager iCubErzelli02 is listed as 2.5 (not 2.5.5), as I guess you verified on the real robot, perhaps we can open an issue in https://github.com/icub-tech-iit/appsAway to ensure that the robot manager entry for iCubErzelli02 is correctly updated? Thanks! 

fyi @vtikha @Fabrizio69

File: README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Both generation pipelines are still in `a work in progress` state, and several i
 |  `YARP_ROBOT_NAME`   | Pipeline     | Notes                           |
 |:--------------------:|:------------:|:-------------------------------:|
 | `iCubDarmstadt01`    | simmechanics | v2.5 without backpack           |
+| `iCubErzelli02`      | simmechanics | v2.5   with backpack            |
```
Comment (traversaro) [inline README.md:38]:
```suggestion
| `iCubErzelli02`      | simmechanics | v2.5.5         |
```

The correct version identified for v2.5 with backpack is v2.5.5, see https://icub-tech-iit.github.io/documentation/icub_versions/ . The other entries are outdated and probably could be updated in another issue.

**Review by traversaro — CHANGES_REQUESTED** [review]
Minor comments.

**Comment by Nicogene** [comment]
It has for sure the backpack but not the battery as far as I know, also without the battery is 2.5.5?

**Comment by traversaro** [comment]
> 
> 
> It has for sure the backpack but not the battery as far as I know, also without the battery is 2.5.5?

I think so. However, if this is not clear from https://icub-tech-iit.github.io/documentation/icub_versions/ we should open an issue in https://github.com/icub-tech-iit/documentation

**Comment by traversaro** [comment]
Note that if there is the backpack, but there is no dummy weight that simulate the presence of the battery, then the difference in weight may affect the torque estimation or any other use of the URDF model that uses inertial information of the `chest` link.

**Comment by traversaro** [comment]
If you prefer we can merge as it is and open issues for tracking the doubts on configuration versions @Nicogene .

**Comment by Nicogene** [comment]
Ok I will try  the wbd device  with the urdf generated and we will see what happens

**Comment by traversaro** [comment]
Ok, please remember to open issues otherwise it is quite easy to forget about this tricky aspects.

**Comment by traversaro** [comment]
> Ok, please remember to open issues otherwise it is quite easy to forget about this tricky aspects.

Hi @Nicogene, just to understand did we opened the issue on the fake iCubErzelli02 backpack?

**Comment by Nicogene** [comment]
Yes here it is #196

**Comment by traversaro** [comment]
> 
> 
> Yes here it is #196

Thanks! For the future let's remember to cross-link related issues so we can find them later.

**Pull request merged**